src/DependencyInjection/EightPointsGuzzleExtension.php

```diff
@@ -150,6 +150,9 @@ protected function createHandler(ContainerBuilder $container, string $clientName
         }
 
         foreach ($this->plugins as $plugin) {
+            if (!isset($options['plugin'][$plugin->getPluginName()])) {
```

Comment:
I would do it the other way round: only execute "loadForClient" if value "isset".

Reply:
Yes, I agree. It will be better.